### ISSUE TITLE
Doesn't work with Beta 3.3

### ISSUE THREAD

**Issue by hellojoeAoPS** [Qantas94Heavy/autopilot-pp#23]
Usually with autopilot, you could click "nav" (beta 3.3), but in the current version of Autopilot++, you don't have an option to do so.

**Comment by Plasmic001** [comment]
Yes I wish someone added this feature. It is the reason I use normal ap over ap++
